symengine/expression.h

```diff
@@ -120,12 +123,23 @@ class Expression
     {
         return eq(*m_basic, *other.m_basic);
     }
+
+    bool operator>=(const Expression &other) const
```

Comment:
Right. Fix for this is not to define `operator>=`, but to change the method in printer itself.
Problem with defining those operators as I mentioned before is that this is not mathematical comparison, but structural comparison and therefore users might get confused.